lib/compat/experimental/blocks.php

```diff
@@ -106,12 +106,24 @@ function get_comments_pagination_arrow( $block, $pagination_type = 'next' ) {
 	 * @return array Filtered editor settings.
 	 */
 	function extend_block_editor_settings_with_discussion_settings( $settings ) {
+
 		$settings['__experimentalDiscussionSettings'] = array(
-			'pageComments'        => get_option( 'page_comments' ),
+			'commentOrder'        => get_option( 'comment_order' ),
 			'commentsPerPage'     => get_option( 'comments_per_page' ),
 			'defaultCommentsPage' => get_option( 'default_comments_page' ),
-			'commentOrder'        => get_option( 'comment_order' ),
+			'pageComments'        => get_option( 'page_comments' ),
+			'threadComments'      => get_option( 'thread_comments' ),
+			'threadCommentsDepth' => get_option( 'thread_comments_depth' ),
+			'avatarURL'           => get_avatar_url(
```

Comment:
I checked the Discussion Settings page and it looks like it has a ton of setting including the default avatar. That was surprising 😄

Reply:
Yep, most surprising is that the default avatar of discussion settings is the same default avatar selected for post authors.